operations/mimir/autoscaling.libsonnet

```diff
@@ -153,7 +153,7 @@
   // `weight` param can be used to control just a portion of the expected queriers with the generated scaled object.
   // For example, if you run multiple querier deployments on different node types, you can use the weight to control which portion of them runs on which nodes.
   // The weight is a number between 0 and 1, where 1 means 100% of the expected queriers.
-  newQuerierScaledObject(name, query_scheduler_container, querier_max_concurrent, min_replicas, max_replicas, target_utilization, weight=1):: self.newScaledObject(name, $._config.namespace, {
+  newQuerierScaledObject(name, query_scheduler_container, querier_container, querier_max_concurrent, min_replicas, max_replicas, target_utilization, weight=1):: self.newScaledObject(name, $._config.namespace, {
```

Comment:
`querier_container_name` sounds more accurate, but should probably change `query_scheduler_container`  too to be consistent. Happy to keep it as it is if you think it's outside of scope

Reply:
We can do it in this PR.

Reply:
Done in https://github.com/grafana/mimir/pull/6971/commits/236cc5cd9be3fe93ce2feda2cb73fae3ff3f3797